src/framework/services/filter/OneOfOption.js

```diff
@@ -0,0 +1,9 @@
+
+export default class OneOfOption {
+
+  constructor(value, label) {
+    this.value = value;
+    this.label = label || value;
+  }
+
+}
```

Comment:
You could write a short test to make sure that `label === value` if non was given in the constructor, but not sure if really needed.